### DISCUSSION TITLE
NPM package

### DISCUSSION THREAD

**Discussion by johannesronaldsson** [PrismJS/live#27] [Ideas]
Even though it's in development, why is this not available through package manager yet? would be great if it were :)a

**Comment by LeaVerou** [comment]
I suppose that makes sense. I'd want to rewrite it to ESM first though, which should happen relatively soon.

**Reply by johannesronaldsson** [reply]
That would be awesome, the world needs a lightweight code editor, I find all the others overkill in general.
